Programming_Problems 5

```diff
@@ -1,5 +1,4 @@
 let n = 17;
 let x = 3 + 6 + 9 + 12 + 15;
-console.log(n);
```

Comment:
I'm sure you can see this isn't correect.

- You're supposed to let the user enter the number n
- The user can enter whatever number like 1000, so you can't manually hardcode (3 + 6 + .9 + ..) like you're doing here.
- What you need is a loop starting from 1 all the way to n, summing the numbers along the way.
- You need to redo problem 4, start from there cos this question builds on that
- Did you go through for/while loops in the programming course

Reply:
Okay sir, I'll work on it sir.
Yes, I went through for/while loops during the programming course.